frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -115,24 +142,27 @@ export const FormComponentConfig = ({
           handleComponentChange={handleComponentUpdate}
         />
       )}
+      {booleanRestPropertyKeys.map((propertyKey) => {
+        if (unsupportedPropertyKeys.includes(propertyKey) || !rest[propertyKey]) return null;
+        return (
+          <EditBooleanValue
+            component={component}
+            handleComponentChange={handleComponentUpdate}
+            propertyKey={propertyKey}
+            key={propertyKey}
+            helpText={rest[propertyKey]?.description}
+          />
+        );
+      })}
 
-      {Object.keys(rest).map((propertyKey) => {
-        if (!rest[propertyKey]) return null;
+      {/** String and number fields (incl. arrays with enum values) */}
+      {remainingRestPropertyKeys.map((propertyKey) => {
+        if (unsupportedPropertyKeys.includes(propertyKey) || !rest[propertyKey]) return null;
```

Comment:
This check is done many times, can we some how filter them away one time in the beginning? 🤔 
Or could we at least wrap the check in a method that is easier to read?